packages/@okta/vuepress-site/docs/guides/custom-url-domain/main/index.md

```diff
@@ -15,10 +15,13 @@ This guide explains how to customize your Okta org with your URL domain name and
 
 **What you need**
 
-For customizing your Okta URL domain:
+For customizing an Okta-managed domain or your own domain:
 
 * A domain that you own, for example, `example.com`
 * A subdomain that you want to use, for example, `login.example.com`
+
+For customizing your own domain only:
```

Comment:
Also here, this is for when customers are providing their own certificate - they are always providing the domain